SparkPost/2web2ui
# SA-2139: Add proper truncation for the compare by aggregate rows

### What Changed

- Added text truncation component w/ tooltip
- Added text truncation to compare by aggregate row for user provided values

### How To Test

- Add compare by filters with long string names (I've found some decent length ones in templates)
- Decrease view port to small view
- Verify that text is properly truncated
- Verify that tooltip exists for truncated text
- Verify that tooltip doesn't exist for non truncated text

### To Do

- [ ] Add test ? Not sure how to easily test this
